docs/sources/tempo/operations/cross_tenant_query.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Cross-tenant query federation
+menuTitle: Query
+description: Cross-tenant query federation
+weight: 70
+aliases:
+- /docs/tempo/operations/cross-tenant-query
+---
+
+
+# Cross-tenant query federation
+
+> NOTE: you need to enable `multitenancy_enabled: true` in the cluster for multi-tenant querying to work.
+> see [enable multi-tenancy]({{< relref "./multitenancy" >}}) for more details and implications of `multitenancy_enabled: true`.
+
+Tempo supports multi-tenant queries. where users can send list of tenants  multiple tenants.
+
+The tenant IDs involved need to be specified separated by a '|' character in the 'X-Scope-OrgID' header.
```

Comment:
Changing to use ` - assuming you want the code formatting around the X-Scope-OrgID header.
```suggestion
The tenant IDs involved need to be specified separated by a '|' character in the `X-Scope-OrgID` header.
```